README.md

<p align="center">
  <img alt="GitHub language count" src="https://img.shields.io/github/languages/count/GabrielBarcelos1/paper-rock-scissors">

  <img alt="Repository size" src="https://img.shields.io/github/repo-size/GabrielBarcelos1/paper-rock-scissors">
  
  <a href="https://github.com/GabrielBarcelos1/paper-rock-scissors/commits/master">
    <img alt="GitHub last commit" src="https://img.shields.io/github/last-commit/GabrielBarcelos1/paper-rock-scissors">
  </a>

  <a href="https://github.com/GabrielBarcelos1/paper-rock-scissors/issues">
    <img alt="Repository issues" src="https://img.shields.io/github/issues/GabrielBarcelos1/paper-rock-scissors">
  </a>

  <img alt="License" src="https://img.shields.io/badge/license-MIT-brightgreen">
</p>

<p align="center">
  <a href="#-tecnologias">🚀 Tecnologias</a>&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;
  <a href="#-projeto">💻 Projeto</a>&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;
  <a href="#-frontend">🌐 Frontend</a>&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;
  <a href="#-mobile(Responsividade)">📱 Mobile</a>&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;
  <a href="#-como-executar">🔖Como Executar</a>&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;
  <a href="#-como-contribuir">🤔 Como contribuir</a>&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;
  <a href="#-licença">🧾 Licença</a>
</p>

<br>

## 🚀 Tecnologias

Esse projeto foi desenvolvido com as seguintes tecnologias:

- [JavaScript]()
- [HTML 5]()
- [Bootstrap]()
- [SweetAlert 2]()
- [Css3]()

## 💻 Projeto
O PROJETO-SA foi o projeto do primeiro semestre do curso Técnico em Desenvolvimento de Sistemas, onde o desafio era desenvolver
um e-commerce, para Desktop e para Mobile, utilizando o armazenamento Web localStorage. O projeto foi desenvolvido por 
uma equipe de 5 integrantes da turma. 



## 🌐 Frontend
<h1 align="center">
    <img  src="https://media.giphy.com/media/4E6Bml9ALTNcnTmGBT/giphy.gif?cid=790b7611a67f64bf56335f694c135f7fdd988d90a386005e&rid=giphy.gif&ct=g" />
</h1>

## 📱 Mobile(Responsividade)
<h1 align="center">
    <img  src="https://media.giphy.com/media/taorFvXxyYVEAZlmH8/giphy.gif?cid=790b761198ac92435acc843e69026b4ddf3758f2d7232f25&rid=giphy.gif&ct=g" />
    <img  src="https://media.giphy.com/media/lhZl92SXid1xcIMMhC/giphy.gif?cid=790b76111b10a94eebdd9c1c679543c4e74c4f4b94575177&rid=giphy.gif&ct=g" />
</h1>

    
## 🔖 Como Executar

#### Clonando o projeto
```sh
git clone https://github.com/thinhague/PROJETO-SA.git
cd PROJETO-SA
Depois disso é so mecher no projeto.
```


## 🤔 Como contribuir

- Faça um fork desse repositório;
- Cria uma branch com a sua feature: `git checkout -b minha-feature`;
- Faça commit das suas alterações: `git commit -m 'feat: Minha nova feature'`;
- Faça push para a sua branch: `git push origin minha-feature`.

Depois que o merge da sua pull request for feito, você pode deletar a sua branch.


## 🧾 Licença

Esse projeto está sob a licença MIT. Veja o arquivo [LICENSE](LICENSE.md) para mais detalhes.

---




jQuery-Mask-Plugin-master/CHANGELOG.md
<!-- == v1.14.16 (Jul/31 2019 16:32 +0100 by Igor Escobar) ==

Bugfixes:

* Fixed oldVal being updated prior to caret position calculation
* Solve loading JQuery as module on Meteor

== v1.14.15 (Mar/08 2018 22:59 +0000 by Igor Escobar) ==

Bugfixes:

* rolling back change to fix caret positioning. it didn’t worked on some devices

== v1.14.14 (Mar/02 2018 16:55 +0000 by Igor Escobar) ==

Bugfixes:

* fixing mask positioning delays
* unmask: also removing place holder if added on the first place.
* unmask: unsetting maxlength if we set it in the first place

== v1.14.13 (Dec/11 2017 18:59 +0000 by Igor Escobar) ==

Bugfixes:

* fixes caret issue explained on #636
* fixing use strict issue

== v1.14.12 (Oct/04 2017 09:57 +0100 by Igor Escobar) ==

Bugfixes:

* bug fixing on caret positioning on some devices

== v1.14.11 (May/30 2017 21:53 +0100 by Igor Escobar) ==

Bugfixes:

* fixing a lot of caret positioning issues. Thanks to @onuradsay

== v1.14.10 (Feb/13 2017 14:18 +0000 by Igor Escobar) ==

Bugfixes:

* fixing exception when oValue in undefined

== v1.14.9 (Jan/25 2017 11:17 +0000 by Igor Escobar) ==

Bugfixes:

* don’t use input event when using samsung browser or old chrome versions

== v1.14.8 (Dec/26 2016 13:18 +0000 by Igor Escobar) ==

Bugfixes:

* fixing caret on android with chrome 28

== v1.14.7 (Dec/25 2016 03:51 +0000 by Igor Escobar) ==

Bugfixes:

* improving caret positioning when cursor is on the middle

== v1.14.6 (Dec/24 2016 17:14 +0000 by Igor Escobar) ==

Bugfixes:

* fix caret positioning with multiple mask chars

== v1.14.5 (Dec/24 2016 14:42 +0000 by Igor Escobar) ==

Changes:

* fixing reserved word

== v1.14.4 (Dec/24 2016 14:38 +0000 by Igor Escobar) ==

Bugfixes:

* fixing android cursor positioning (special thanks to @felipejunges and @fernandobandeira)

== v1.14.3 (Nov/28 2016 11:53 +0000 by Igor Escobar) ==

Bugfixes:

* fixing caret positioning on safari

== v1.14.2 (Nov/27 2016 20:04 +0000 by Igor Escobar) ==

Bugfixes:

* apply auto maxlength in case the mask doesn't have recursive pattern

== v1.14.1 (Nov/27 2016 19:20 +0000 by Igor Escobar) ==

Bugfixes:

* Fix input value mangling when inserting before a static mask character
* fixing caret position issue

== v1.14.0 (Apr/03 2016 17:52 +0100 by Igor Escobar) ==

Bugfixes:

* Fix cursor jumping while editing in non-IE browsers. Thanks to @archwyrm

Features:

* adding masked function for better angular use

== v1.13.9 (Mar/20 2016 16:17 +0000 by Igor Escobar) ==

Changes:

* giving the opportunity to pass watchInputs locally

== v1.13.8 (Mar/06 2016 23:25 +0000 by Igor Escobar) ==

Changes:

* adding support for meteor

== v1.13.7 (Mar/06 2016 22:46 +0000 by Igor Escobar) ==

Bugfixes:

* fixing onChange behaviour

== v1.13.6 (Mar/06 2016 22:14 +0000 by Igor Escobar) ==

Bugfixes:

* fixing deploy procedure

== v1.13.5 (Mar/06 2016 22:01 +0000 by Igor Escobar) ==

Changes:

* adding clearIfNotMatch to globalOptions

Bugfixes:

* fixing some bugs when using non-input elements
* fixing mobile issues at #348.
* using input event when supported

== v1.13.4 (Aug/07 2015 14:21 +0100 by Igor Escobar) ==

Bugfixes:

* Add check to ensure that there are input elements before using them

== v1.13.3 (Jul/16 2015 16:11 +0100 by Igor Escobar) ==

Changes:

* adding main property to package.json

== v1.13.2 (Jul/16 2015 16:06 +0100 by Igor Escobar) ==

Bugfixes:

* change event wasnt being triggered in some cases

== v1.13.1 (Jul/07 2015 15:38 +0100 by Igor Escobar) ==

Bugfixes:

* destroying input event too

== v1.13.0 (Jul/07 2015 15:26 +0100 by Igor Escobar) ==

Changes:

* removing the autocomplete default.

Bugfixes:

* fixing bower file thanks to @lazyants

Features:

* prevent glitch when invalid chars.
* turning off autocomplete when browsers doesn't support oninput event.

== v1.12.0 (Jul/07 2015 11:37 +0100 by Igor Escobar) ==

Features:

* giving an alternative to the autocomplete/autofill problem.

== v1.11.4 (Feb/26 2015 22:11 +0000 by Igor Escobar) ==

Changes:

* grunt, jshint and better applyDataMask. Thanks to @lagden
* automated deploy to npm

== v1.11.3 (Jan/28 2015 15:41 +0000 by Igor Escobar) ==

Changes:

* Added commonjs module definition

== v1.11.2 (Dec/26 2014 15:36 +0000 by Igor Escobar) ==

Bugfixes:

* unreachable code

== v1.11.1 (Dec/26 2014 15:34 +0000 by Igor Escobar) ==

Bugfixes:

* unreachable code

== v1.11.0 (Dec/26 2014 15:33 +0000 by Igor Escobar) ==

Features:

* implementing selectOnFocus and data-mask-selectonfocus option
* adding public method called: .applyDataMask in case you want to decide whether to apply masks in data-mask fields

== v1.10.13 (Nov/19 2014 16:06 +0000 by Igor Escobar) ==

Bugfixes:

* fixing bug with watchInputs feature when mask is used as a function and not a string.

== v1.10.12 (Nov/06 2014 13:08 +0000 by Igor Escobar) ==

Changes:

* making a few improvements to make selection, copy events easier

== v1.10.11 (Nov/06 2014 11:26 +0000 by Igor Escobar) ==

Bugfixes:

* we need to revaluate dataMask flags everytime

== v1.10.10 (Nov/06 2014 10:41 +0000 by Igor Escobar) ==

Bugfixes:

* fixing dynamically data-mask added elements

== v1.10.9 (Nov/05 2014 10:52 +0000 by Igor Escobar) ==

Bugfixes:

* data-mask wasnt working

== v1.10.8 (Nov/01 2014 13:49 +0000 by Igor Escobar) ==

Changes:

* we dont need to seek for data-mask every time

== v1.10.7 (Nov/01 2014 13:18 +0000 by Igor Escobar) ==

Changes:

* little optimization

== v1.10.6 (Oct/28 2014 13:59 +0000 by Igor Escobar) ==

Bugfixes:

* fixing weird cursor problems in weird cases.
* dynamically added inputs wasnt working

== v1.10.5 (Oct/23 2014 11:41 +0100 by Igor Escobar) ==

Bugfixes:


js/canecas.js


produtos = JSON.parse(localStorage.getItem('produtos'))


let pags = 0

let passarPagina = document.querySelector('.passar-pagina')
let a = passarPagina.querySelectorAll('a')
a[0].style = 'opacity: 0.6'

function botbot2() {
    vitrineGo(12)
   
    a[0].style = 'opacity: 1'
    a[1].style = 'opacity: 0.6'
    a[2].style = 'opacity: 1'
}

function botbot3() {
    vitrineGo(24)
   
    a[0].style = 'opacity: 1'
    a[1].style = 'opacity: 1'
    a[2].style = 'opacity: 0.6'
}


function vitrineGo(pag) {
    console.log(produtos.length);

    document.getElementById('vitTotal').innerHTML = ``

    // cria a quantidade de divs (linhas) necessárias

    let quantDivs = Math.ceil(produtos.length / 4)

    for (let i = 1; i <= quantDivs; i++) {
        document.getElementById('vitTotal').innerHTML += `
            <div id="vitProd${i}" class="container1">  </div>
        `
    }

    // adiciona os produtos

    pags = pag

    let j = 1

    for (let i = pags; i < produtos.length; i++) {


        document.getElementById(`vitProd${j}`).innerHTML += `

            <div class="produtos"  style="background-image: url(../img/canecas-disney/ariel tras.png);">             
                <img class="tete1" onclick="caneca(${produtos[i].id})" src="${produtos[i].foto_produto}">                   
                <label for="">${produtos[i].nome}<br>${produtos[i].preco}</label>
                <a href="javascript:add_car(${produtos[i].id}) , msg_add();" >Adicionar ao carrinho</a>
            </div>     
        `
        if (i == 3 || i == 7 || i == 15 || i == 19 || i == 27 || i == 31) {
            j++
        }
        if (i == 11) {
            return false
        }
        if (i == 23) {
            return false
        }
    }

}
